components/brave_extension/extension/brave_extension/types/state/shieldsPannelState.ts

```diff
@@ -43,6 +44,7 @@ export interface State {
   currentWindowId: number
   tabs: Tabs
   windows: Windows
+  theme: ThemeType
```

Comment:
Should we really add a `theme` prop to specific state for each UI we have? I'm not sure and think it's adding noise and complication. The data reducers don't depend on this property at the moment, and if we do want it on state, we can use a shared store/reducer that can be imported for any page which wants it, e.g. UIThemeStore, combined via `combineReducers`.